gyp/.github/workflows/Python_tests.yml

```diff
@@ -0,0 +1,31 @@
+# TODO: Enable os: windows-latest
+# TODO: Enable python-version: 3.5
+# TODO: Enable pytest --doctest-modules
+
+name: Python_tests
+on: [push, pull_request]
+jobs:
+  Python_tests:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      max-parallel: 15
+      matrix:
+        os: [macos-latest, ubuntu-latest] # , windows-latest]
+        python-version: [2.7, 3.6, 3.7, 3.8] # 3.5,
+    steps:
+      - uses: actions/checkout@v1
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements_dev.txt
+      - name: Lint with flake8
+        run: flake8 . --count --show-source --statistics
```

Comment:
Be explicit by putting ALL options here.  Having some options here and others in a dot file bugs me.